packages/react/src/primitives/types/passwordField.ts

```diff
@@ -57,6 +57,12 @@ export interface PasswordFieldProps extends TextFieldProps {
    * Forwarded ref for access to show password button DOM element
    */
   showPasswordButtonRef?: React.Ref<HTMLButtonElement>;
+
+  /**
+   * @description
+   * Indicates that Field is in error state.
+   */
+  hasError?: boolean;
```

Comment:
`TextFieldProps` extends `InputProps` which has this prop, so this line should not be needed.

Reply:
okay